Core/Src/main.c

```diff
@@ -74,7 +78,46 @@ int main(void)
   HAL_Init();
 
   /* USER CODE BEGIN Init */
-
+  	  // pow(2,10)-1 because of 10 bits, considering pot response 0-1023
+	  const int DUTY_CYCLE_COUNTS_5_PERCENT = 3200;
+
+	  /* I set the prescaler to 14, and the total count period
+		   * to 64000.
+		   * To convert pot response (0-1023) to duty cycle counts
+		   * (3200-6400), the values must correspond in this fashion:
+		   *
+		   * 0 => 3200, and 1023 => 6400
+		   *
+		   * This should follow a standard formula. If 0 corresponds to
+		   * 3200, that means the formula to convert pot response to
+		   * counts must include a scalar addition of 3200. Followign a
+		   * mx + b fashion, the multiplier must be as follows:
+		   *
+		   * 6400 = m(1023) + 3200
+		   * or that m = 3.128f*/
+  	  const double SCALING_FACTOR = 3.128;
+
+  	  //Timeout for spi transmit receive function in ms
+  	  const uint32_t SPI_TIMEOUT = 250;
+  	  const uint8_t NUM_BITS_TO_SEND = 3;
+  	  //Bit mask as we only need the last 2 bits in the second returned byte
+  	  const uint8_t BIT_MASK = 0x00000011;
```

Comment:
I see what you are trying to do here but remember by adding a `0x` at the beginning, you are indicating that the following value is a hex value meaning 0x11 is actually 1011 in binary, which I believe is not what you want.